### DISCUSSION TITLE
Deployer process hangs with no terminal output

### DISCUSSION THREAD

**Discussion by imjb87** [deployphp/deployer#2278] [Help needed]
Might anyone know why deployer has suddenly become unresponsive today? It's happened on all my sites (20-30 sites). The process still seems to run, but it hangs in the terminal and doesn't output any verbose information. See below:

![Capture](https://user-images.githubusercontent.com/46000619/98568344-e5c9f480-22a8-11eb-9b93-a421327798c8.PNG)

**Comment by antonmedv** [comment]
What deployer version?

**Comment by imjb87** [comment]
6.8. When I run it with -vvv it returns this: "fatal: Not a git repository (or any of the parent directories): .git". However it continues to checkout the files and then hangs.

**Comment by antonmedv** [comment]
What deployment OS target?
